tests/integration/backward_compatible/reconnect_test.py

```diff
@@ -268,3 +280,6 @@ def assertion():
             self.assertNotEqual(0, event_count.get())
 
         self.assertTrueEventually(assertion)
+
+        client.shutdown()
+        self.rc.terminateCluster(cluster.id)
```

Comment:
how was the client shutting down before this added?

Reply:
It was not :D That was the problem

Reply:
ok that question was to clarify 😄 , thanks